docs/reference/ExifTags.rst

```diff
@@ -7,7 +7,8 @@
 The :py:mod:`ExifTags` module exposes two dictionaries which
 provide constants and clear-text names for various well-known EXIF tags.
 
```

Comment:
Looks like everything after this is missing?

![image](https://user-images.githubusercontent.com/1324225/85229475-8602a800-b3f2-11ea-9db1-fb4ad1ce5913.png)


https://pillow--4697.org.readthedocs.build/en/4697/reference/ExifTags.html

# master

![image](https://user-images.githubusercontent.com/1324225/85229499-9dda2c00-b3f2-11ea-8d96-17f619e86371.png)

https://pillow.readthedocs.io/en/stable/reference/ExifTags.html

Reply:
I had Sphinx 3.0.1 installed locally. Looking at the [build page](https://readthedocs.org/projects/pillow/builds/11294859/), I can see that `sphinx<2` is specified, but I can't find any references to this in the repo. Downgrading also breaks the page for me. Do you know where this is coming from?

`/home/docs/checkouts/readthedocs.org/user_builds/pillow/envs/4697/bin/python -m pip install --upgrade --no-cache-dir Pygments==2.3.1 setuptools==41.0.1 docutils==0.14 mock==1.0.1 pillow==5.4.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<1.1`

This line is also installing an old version of Pillow, only to recompile it in the next step.

Reply:
Found it here: https://docs.readthedocs.io/en/stable/intro/import-guide.html#building-your-documentation
> It is also important to note that the default version of Sphinx is v1.8.5. If choosing to build your documentation other than this, it must be specified in a requirements.txt file.

Is there any reason not to upgrade?

Either way, this differs from the version used to test docs on CIs ([typically latest](https://github.com/python-pillow/Pillow/runs/792837171?check_suite_focus=true#step:13:10)).

Reply:
Good find, we should probably upgrade.

Reply:
I added `sphinx>2.4` (the lowest version with `:type:` for `:data` fixing this error) to requirements in the most recent commit, which is installing latest for now (until RTD update their default). I'm not sure if there is a better way to stay on latest.